src/test/librados_test_stub/TestIoCtxImpl.h

```diff
@@ -91,6 +91,7 @@ class TestIoCtxImpl {
                                AioCompletionImpl *c, int flags,
                                bufferlist *pbl);
   virtual int aio_remove(const std::string& oid, AioCompletionImpl *c) = 0;
+  virtual int aio_remove(const std::string& oid, AioCompletionImpl *c, int flags) = 0;
```

Comment:
Nit: probably cleaner to just add ```int flags``` to the previous one and update ```IoCtx::aio_remove``` in  ```LibRadosTestStub``` to pass zero if it's the version w/o the flags param.